code/modules/mob/living/carbon/human/life.dm

```diff
@@ -94,14 +94,13 @@ var/global/list/organ_damage_overlays = list(
 // Doing this during species init breaks shit.
 /mob/living/carbon/human/proc/DeferredSpeciesSetup()
```

Comment:
This is all undoing a small fix that aimed solely to make species spawn with their genetic powers, I now realize it didn't go far enough and made a 'true' fix.